TabulateSmarterTestContentPackage/Tabulator.cs

```diff
@@ -2289,6 +2329,16 @@ static string ToDependsOnString(string itemId, string dependsOnId)
             return string.Concat(itemId, "~", dependsOnId);
         }
 
+        internal bool ReportMissingImgAltTags(string input)
+        {
+            const string imgAltMatcherPattern = @"<\s*img[^>]*>";
+            var result = Regex.Match(input, imgAltMatcherPattern);
+
+            const string imgAltPattern = @"alt\s*=\s*['""]\s*\w+\s*['""]";
+            return result.Success
+                && !Regex.Match(result.Value, imgAltPattern).Success;
```

Comment:
I just saw a recent commit that uses an HtmlReader to parse HTML instead of RegEx.  Can you take a look at it and see if it feels cleaner or not since it is used elsewhere in the program?

https://github.com/SmarterApp/TabulateSmarterTestContentPackage/commit/a91efeb91de6d3480931a13a232d1d27b8b9dd12#diff-73eb2e803faded1728e51e14ac135bd2R1456